src/user-points/user-points.jobs.controller.ts

```diff
@@ -68,10 +74,69 @@ export class UserPointsJobsController {
     }
 
     const options = await this.eventsService.getUpsertPointsOptions(user);
-    await this.userPointsService.upsert(options);
+    const userPoints = await this.userPointsService.upsert(options);
+
+    if (
```

Comment:
Won't these default to 0 based on last PR?

Reply:
They are nullable: https://github.com/iron-fish/ironfish-api/pull/1275/files#diff-5b443964f4f3a611682db8f7e02177b0a8c632b2039e2bd5e4dd7347815c565cR265-R267